emcc.py

```diff
@@ -540,7 +540,7 @@ def build_symbol_list(filename):
     """
     library_syms = generate_js_sym_info()
 
-    write_file(filename, json.dumps(library_syms, separators=(',', ':')))
+    write_file(filename, json.dumps(library_syms, separators=(',', ':'), indent=2))
```

Comment:
Does this affect build time? I just worry about it creating a much larger json file, which could run into disk I/O speed issues. If so this could be only in debug mode perhaps? But I guess I'm not sure how big this file even is.

Reply:
I can't imagine this effecting build times.   

This file is ~1600 lines line and about 43k in size.   So that newlines add very little, and I can't believe if parse time differs.  If serialization time is significant we should stop using json here and just use comma separated data.  But I'm sure its not the process that generates this file is the processing of all JS library files which takes ~100ms IIRC